cypress-io/cypress
# feat: Add svelte detection to wizard

- Closes #23432

### User facing changelog


### Additional details


### Steps to test

1. Open launchpad
1. Select a svelte project
1. Select `Component Testing`
1. Wizard should detect Svelte and correct bundler
1. Click `Next Step`
1. Wizard should correctly validate that required dependencies are installed
2. Click `Continue`
3. The following criteria should be met in the `Configuration Files`
    - `cypress.config.js` should have a devServer set up as shown below:
    ```js
    devServer: {
        framework: "svelte",
        bundler: "vite",
      },
    ```
    - `cypress/support/component.js` should include the correct mount
      ```js 
      import { mount } from 'cypress/svelte'`
      ```

### How has the user experience changed?


### PR Tasks


- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?
